src/app/clusters/push-av-stream-transport-server/push-av-stream-transport-logic.cpp

```diff
@@ -541,6 +541,40 @@ PushAvStreamTransportServerLogic::HandleAllocatePushTransport(CommandHandler & h
     Commands::AllocatePushTransportResponse::Type response;
     auto & transportOptions = commandData.transportOptions;
 
+    IngestMethodsEnum ingestMethod = commandData.transportOptions.ingestMethod;
+
+    bool isFormatSupported = false;
+
+    for (auto & supportsFormat : mSupportedFormats)
+    {
+        if ((supportsFormat.ingestMethod == ingestMethod) &&
+            (supportsFormat.containerFormat == commandData.transportOptions.containerOptions.containerType))
+        {
+            isFormatSupported = true;
+        }
+    }
+
+    if (isFormatSupported == false)
+    {
+        auto status = to_underlying(StatusCodeEnum::kInvalidCombination);
+        ChipLogError(Zcl,
+                     "HandleAllocatePushTransport[ep=%d]: Invalid Ingest Method and Container Format Combination : (Ingest Method: "
+                     "%02X and Container Format: %02X)",
+                     mEndpointId, to_underlying(ingestMethod),
+                     to_underlying(commandData.transportOptions.containerOptions.containerType));
+        handler.AddClusterSpecificFailure(commandPath, status);
+        return std::nullopt;
```

Comment:
Those three non-log lines might be a _lot_ more readable as:
```
    return ClusterStatusCode::ClusterSpecificFailure(StatusCodeEnum::kInvalidCombination);
```

no?  Similar in some other places here.  But I don't feel too strongly about this.